shared/charts/gpii-flowmanager/templates/virtual_service_googleapis.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.enableStackdriverTrace }}
```

Comment:
`VirtualService` is not needed to allow access to external service, only the `ServiceEntry` -> remove

Reply:
Same here.